portal/server/controllers/dashboard/deals/template-filters.js

```diff
@@ -9,6 +9,19 @@ const {
   submissionTypeFilters,
 } = require('../filters/generate-template-filters');
 
+const createdByYouFilter = (submittedFilters) => {
+  const fieldName = FIELD_NAMES.DEAL.CREATED_BY;
+
+  const fieldInputs = [
+    {
+      text: CONTENT_STRINGS.DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.DEALS.CREATED_BY_YOU,
```

Comment:
can we deconstruct these lines? it gets very long otherwise

Reply:
done